Makefile

```diff
@@ -253,3 +253,11 @@ release-nightly:
 component-images:
 	cd packaging/images/components && ./build.sh
 .PHONY: component-images
+
+licensecheck: DEBUG=true
```

Comment:
Ce we remove the DEBUG statement? Otherwise, it generates too much output.

Reply:
That was really to force debug info on the microshift build, as lichen analyzes the binary and not the source code ( a bit weird if you ask me).  

I checked and it works without that DEBUG, so I will remove it for simplicity.